### ISSUE TITLE
Feature request: easily access context / application properties in EL

### ISSUE THREAD

**Issue by jepsar** [omnifaces/omnifaces#311]
I was writing this expression in a `rendered` attribute: `#{facesContext.application.projectStage == 'Development'}`. Then I remembered [`Faces#isDevelopment()`](https://github.com/omnifaces/omnifaces/blob/2.5/src/main/java/org/omnifaces/util/Faces.java#L282) and wondered if OmniFaces also offered a way to easily access this in EL. Could not find anything. In my project I'll be creating a bean to expose `Faces#isDevelopment()`.

**Comment by BalusC** [comment]
One way is [`<o:importFunctions>`](http://showcase.omnifaces.org/taghandlers/importFunctions).

```
<o:importFunctions type="org.omnifaces.util.Faces" />
<h;someComponent rendered="#{Faces:isDevelopment()}" />
```

I however do agree that something like a `#{faces.development}` would be useful. This is definitely possible with a custom `ELResolver` which delegates further to `Faces` utility class, so I'll keep the issue open.

**Comment by jepsar** [comment]
Thanks! +1 for `o:importFuctions` I'll be using that.

**Comment by BalusC** [comment]
`#{faces}` is now available with among others `#{faces.development}`.

It's available in today's latest [2.6-SNAPSHOT](https://oss.sonatype.org/content/repositories/snapshots/org/omnifaces/omnifaces/2.6-SNAPSHOT/)
